### PR TITLE
feat : added required code

### PR DESCRIPTION
**Pull request by Dishika18**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,18 +17,29 @@ Sample description
 1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
 2. **Create the Workspace**:  
    ```bash  
-   daytona create <SAMPLE_REPO_URL> 
-   ```  
+   daytona create https://github.com/Dishika18/CrisisMonitor-Daytona.git
```
Comment (Tpuljak) [inline README.md]: Link is wrong, should be https://github.com/daytonaio/sample-django-crisis-monitor

P.S. I realized that I named the repo `crisis-montior`, I'll rename it after we merge

**Review by Tpuljak — CHANGES_REQUESTED** [review]
Please add a sign off to your commit.
You can do it like so:
```
git rebase HEAD~1 --signoff
git push -f
```

**Comment by Dishika18** [comment]
> Please add a sign off to your commit. You can do it like so:
> 
> ```
> git rebase HEAD~1 --signoff
> git push -f
> ```

@Tpuljak It's Done!

File: README.md (hunk 1 above)
Comment (Dishika18) [reply README.md]: Ok so once the repo is renamed, should I update the README with the new name, or is that not necessary?

Comment (Tpuljak) [reply README.md]: Update the command to `daytona create https://github.com/daytonaio/sample-django-crisis-monitor` now and I'll change the name later.

Comment (Dishika18) [reply README.md]: Ok, I have updated the readme.

Comment (Tpuljak) [reply README.md]: @Dishika18 please set it exactly as `daytona create https://github.com/daytonaio/sample-django-crisis-monitor`

**Pull request merged**